workflow/metrics/build_info_test.go

```diff
@@ -0,0 +1,32 @@
+package metrics
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/otel/attribute"
+
+	"github.com/argoproj/argo-workflows/v3"
+)
+
+func TestBuildInfo(t *testing.T) {
+	_, te, err := CreateDefaultTestMetrics()
+	if assert.NoError(t, err) {
```

Comment:
could invert the conditional here as well

Reply:
I'm not sure why we'd do that.
Anyway, switched to `require` instead for these.